integrations/terraform/envs/portal/config/data_glossary/data_outputs.yaml

```diff
@@ -1,17 +1,20 @@
 raw_test:
-  s3:
-    - bucket_name: datalake
-      path: /raw/test
-  owner: ["test-project-1"]
+  S3:
+    bucket: datalake
+    prefix: /raw/test
+  owner: test-project-1
```

Comment:
Trust me, you will end up with multiple ownerships...

Reply:
This would mean allowing multiple data products to have write permissions on the same data output, is this desirable?

Reply:
As discussed. Given that the frontend and backend currently only allow single owners, I would leave it like this for now